packages/bpk-scrim-utils/src/BpkScrim.js

```diff
@@ -17,20 +17,36 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { TransitionInitialMount, cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-scrim.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkScrim = () => (
+const BpkScrim = props => (
+  /* eslint-disable jsx-a11y/no-static-element-interactions */
+  /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Can you exapand on why we have disabled these two rules? I'm finding the ESC still works to dismiss, why is this given that we've removed key events?

Reply:
It's actually the `Portal` component that's responsible for the escape behaviour. The scrim itself does not handle the Escape key-press.

Reply:
So `click-events-have-key-events` is disabled as we are infact providing this already (through the `Portal`'s key listener,
As for `no-static-element-interaction` - I don't know if there is a suitable role to assign to the scrim as it's bascically just a visual element

Reply:
Cool seems fair then